src/appengine/handlers/upload_testcase.py

```diff
@@ -165,6 +166,17 @@ def filter_target_names(targets, engine):
   return [t[len(prefix):] for t in targets if t.startswith(prefix)]
 
 
+def filter_blackbox_fuzzers(fuzzers):
+  """Filter out fuzzers such that only blackbox fuzzers are included."""
+
+  def is_greybox_fuzzer(name):
+    # TODO(ochang): Generalize this check.
+    return (name.startswith('libFuzzer') or name.startswith('afl') or
```

Comment:
Do we have a constant anywhere else that would cover these three? This seems like something we'd easily forget to update when new things are added.

Reply:
i thought we had a builtin_fuzzers that we can check for names ?

Reply:
It's a mess right now because of engine vs builtin fuzzers. I will clean this all up once AFL is migrated.